importlib_metadata/compat/stdlib.py

```diff
@@ -0,0 +1,27 @@
+"""
+Compatibility layer with stdlib.
+Only needed when distributing via PyPI/3rd-party package.
+"""
+
+import sys
+from typing import TYPE_CHECKING, List, Union
+
+if TYPE_CHECKING:
+    # Avoid circular imports
+
+    from importlib import metadata as _legacy
+
+    from typing_extensions import TypeAlias
+
+    from .. import Distribution, PackagePath, _meta
```

Comment:
Shouldn't these come from `importlib.metadata`?

Reply:
Aah. Okay. Yes.

The term legacy was throwing me off. I was thinking surely `importlib_metadata` would be the legacy. Thanks for confirming.